docs/excel/custom-functions-overview.md

```diff
@@ -17,7 +17,7 @@ function ADD42(a, b) {
 Custom functions are now available in Developer Preview on Windows, Mac, and Excel Online. Follow these steps to try them:
 
 1.  Install Office (build 9325 on Windows or 13.329 on Mac) and join the [Office Insider](https://products.office.com/office-insider) program. (Note that it isn't enough just to get the latest build; the feature will be disabled on any build until you join the Insider program)
-2.  Clone the [Excel-Custom-Functions](https://github.com/OfficeDev/Excel-Custom-Functions) repo and follow the instructions in the README.md to start the add-in in Excel, make changes in the code, and debug.
+2.  Create an Excel-Custom-Functions using Yo Office (https://github.com/OfficeDev/generator-office), and follow the instructions in the project README.md to start the add-in in Excel, make changes in the code, and debug.
```

Comment:
Instead of "an Excel-Custom-Functions", consider using the text from yo office ("Excel Custom Functions Add-in project").

Reply:
Now using "Excel Custom Functions Add-in project"